cf-rd/src/main/java/org/eclipse/californium/tools/resources/RDNodeResource.java

```diff
@@ -49,7 +50,7 @@
 	 * if it does not respond.
 	 */
 	private static ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor(//
-			new Utils.DaemonThreadFactory("RDLifeTime#"));
+			new Utils.DaemonThreadFactory());
```

Comment:
There is no default constructor defined on Californium master. We need to keep it like it is ...